### PR TITLE
Categorize embeds for easier discovery

### PR DESCRIPTION
**Pull request by notnownikki**

### CODE REVIEW

File: blocks/api/categories.js [hunk 1]
```diff
@@ -16,7 +16,12 @@ const categories = [
 	{ slug: 'formatting', title: __( 'Formatting' ) },
 	{ slug: 'layout', title: __( 'Layout Blocks' ) },
 	{ slug: 'widgets', title: __( 'Widgets' ) },
-	{ slug: 'embed', title: __( 'Embed' ) },
+	{ slug: 'embed-common', title: __( 'Common' ) },
```
Comment (mtias) [inline blocks/api/categories.js]: Do we need "common"?

Comment (notnownikki) [inline blocks/api/categories.js]: Probably not... Although this is just for getting feedback on the UX at this point :)

**Comment by annaephox** [comment]
Hello again @notnownikki! Any chance you could add a screenshot to this issue... I'm sorry for my lame designer head that is afraid of reading code :-)

**Comment by karmatosed** [comment]
I really like this @notnownikki. My only wonder is 'Blogs and Documents' and if there is something better to categorise there. I think it's worth merging though (once the conflict has been resolved).

For reference this is what it looks like:

2017-08-25 at 21 59

**Comment by notnownikki** [comment]
@karmatosed thank you for screenshotting that. @annaephox I'm really sorry for not putting up screenshots, it was exclusionary of me, and I apoligize for making it difficult for you to give feedback. I won't make that mistake again. :heart:

I'll clean up the conflicts on those branches and gif/screenshot next week after the public holiday here :)

**Comment by youknowriad** [comment]
@notnownikki Any plans for this PR?

**Comment by notnownikki** [comment]
Unless anyone thinks there's a pressing need for this, I'm happy to close it. I think adding the extra search keywords to blocks makes them easier to find and I haven't seen anyone wanting to revisit this in a while.

**Comment by youknowriad** [comment]
Ok, let's close for now and see if it comes up again. Thanks @notnownikki

**Pull request closed without merging**